packages/cli/src/templates/Canvas/index.tsx

```diff
@@ -86,19 +86,24 @@ const Page = () => {
   const { selectedTable, openedTables, setOpenedTables, setSelectedTable } =
     useCanvasContext();
 
-  const bottomTabs = useMemo(
+  const bottomTabs: HvCanvasBottomPanelProps["tabs"] = useMemo(
     () =>
       openedTables?.map((table) => ({
         id: table.id,
-        title: (
-          <div className={classes.titleRoot}>
-            <Table />
-            <div className={classes.titleContainer}>
+        title: (overflowing) =>
```

Comment:
Is this the extra `<div>` & class to enforce the margin of the icons? I think it's better if we don't do that - we get a few more pixels for text and this only happens in an "extreme" scenario.

We could simplify it to something like this
```tsx
(overflowing) => (
  <div className={classes.titleRoot}>
    {!overflowing && <Table />}
    <HvOverflowTooltip data={table.label} />
  </div>
)
```

![image](https://github.com/user-attachments/assets/37c14c75-c6ac-4408-823d-dc06da8656d2)

Also, the `marginRight` on the `rightActions` coming from `HvActionsGeneric` looks a bit excessive. This margin was intended for normal buttons

Reply:
I'll update this